README.md

```diff
@@ -11,6 +11,11 @@ Code coverage: (develop branch)
 [![Coverage Status](https://coveralls.io/repos/github/kbase/catalog/badge.svg?branch=develop)](https://coveralls.io/github/kbase/catalog?branch=develop)
 
 
+#### v2.1.2 - 3/16/18
+  - Pull a new base image if possible each time a module is registered
+  - Fix extra_files logic
+  - Update NMS to use Auth2
```

Comment:
Since this only affects testing, I don't think it should be included in release notes - confusing otherwise